build.cake

```diff
@@ -134,6 +134,20 @@ Task ("binderate")
 Task("libs")
 	.Does(() =>
 {
+	if (bool.TryParse(EnvironmentVariable("PRE_RESTORE_PROJECTS") ?? "false", out var restore) && restore) {
+		var restoreSettings = new MSBuildSettings()
+			.SetConfiguration(CONFIGURATION)
+			.SetVerbosity(VERBOSITY)
+			.SetMaxCpuCount(0)
+			.WithProperty("DesignTimeBuild", "false")
+			.WithProperty("AndroidSdkBuildToolsVersion", "28.0.3")
+			.WithTarget("Restore");
+
+		foreach (var csproj in GetFiles("./generated/**/*.csproj")) {

```

Comment:
For some reason, a global restore is broken. So, just restore each project be fore building.

Reply:
Got it.